core/database_api.php

```diff
@@ -743,7 +743,6 @@ function db_prepare_binary_string( $p_string ) {
 
 	switch( $t_db_type ) {
 		case 'mssql':
-		case 'mssqlnative':
```

Comment:
To make things more obvious, I would move the _mssqlnative_ case before the _oci8_ one instead of removing it, and update the fall-through comment accordingly.